### PR TITLE
Add Atom Pair docs and fixes

### PR DESCRIPTION
**Pull request by j-adamczyk**

### CODE REVIEW

File: docs/make.bat
Comment (mjste) [inline docs/make.bat:1]: maybe we should leave CRLF, as .bat files are meant for windows

File: docs/_templates/class.rst
Comment (mjste) [inline docs/_templates/class.rst:1]: move it to _templates/autosummary/class.rst
Without it, the file does not affect anything

File: docs/_templates/class.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+.. autoclass:: {{ objname }}
+   {% block methods %}
+
+   {% if methods %}
+   .. rubric:: {{ _('Methods') }}
+
+   .. autosummary::
+   {% for item in methods %}
+      ~{{ name }}.{{ item }}
```
Comment (mjste) [inline docs/_templates/class.rst:13]: please add 
`
{% if item != '__init__' %} 
  ~{{ name }}.{{ item }}
{% endif %}
`

**Review by mjste — CHANGES_REQUESTED** [review]
at least class.rst must be moved

File: docs/make.bat
Comment (j-adamczyk) [reply docs/make.bat:1]: done

File: docs/_templates/class.rst (hunk 1 above)
Comment (j-adamczyk) [reply docs/_templates/class.rst:13]: done

**Review by mjste — APPROVED** [review]

**Pull request merged**